definitions/marts/npq_separation_prep/tad_npq_participant_declarations.sqlx

```diff
@@ -19,7 +19,8 @@ config {
         lead_provider_name: "The name of the Lead Provider who submitted the declaration.",
         declaration_date: "Date evidence was received.",
         declaration_type: "The declaration type which for NPQ can be started, retained-1, retained-2 or completed.",
-        application_id: "ID used to link to NPQ Enrolments (application_id)",
+        // application_id: "ID used to link to NPQ Enrolments (application_id)",
+        application_ecf_id: " GUID for the application to link to NPQ Enrolments (application_id)",

```

Comment:
I don't think this description is sufficiently clear for TAD. I don't think GUID in general is sufficiently descriptive to explain how NPQ are using it (in any mart let alone a TAD mart).